README.md

```diff
@@ -24,8 +24,15 @@ git clone --branch=<<branch>> --recurse-submodules https://github.com/nightscout
 cd Trio
 ```
 
-modify Config.xconfig to replace ##TEAM_ID## with your Apple Developer ID.
+Create an ConfigOverride.xcconfig file that will automate signing of the build targets in Xcode:
 
+```
+echo 'DEVELOPER_TEAM = xxxxxxxxxx' > ConfigOverride.xcconfig
+```
+
+Modify the Config.xconfig to replace `xxxxxxxxxx` with your Apple Developer ID. You can do this from within Xcode, or by using some other text editor.
```

Comment:
How about recommending `nano ConfigOverride.xcconfig` instead of "some other text editor", since it is CLI instructions.

Reply:
Also, maybe recommend replacing xxxxxxxxxx with your TEAMID _before_ entering the command in Terminal.

Reply:
I was thinking that involving the nano editor would be too complicated, would have to instruct about saving the file (ctrl-o) and existing nano (ctrl-x). Which is possible of course. Is nano always shipped with Mac by the way? I think the default editor might be `vi` 😬

Reply:
But instructing to replace xxxxxx with the teamID before pasting the command into Terminal is probably the easiest and most lightweight solution. And if someone gets it wrong, it's just to repeat it, as `>` will replace the line as opposed to append it. Or one can edit in Xcode.

Reply:
I pushed that modification now.

Reply:
All hail nano! I do believe it is shipped with MacOS by default, they even have a support page on it (admittedly slightly to my surprise.).